api/v1alpha1/virtualmachinesetresourcepolicy_types.go

```diff
@@ -51,6 +51,7 @@ type VirtualMachineSetResourcePolicyStatus struct {
 type ClusterModuleStatus struct {
 	GroupName  string `json:"groupname"`
 	ModuleUuid string `json:"moduleUUID"`
+	Cluster    string `json:"cluster"`
```

Comment:
Please rename this to `ClusterMoId` and `clusterMoId` in the JSON tag. Because the example value provided is the managed object ID for a cluster.